BSP/STM32F413/Src/how_to_run_gdb.txt

```diff
@@ -0,0 +1,8 @@
+run `st-util`
+
+run `gdb-multiarch Objects/control-leader.elf`
+
+> target extended-remote localhost:4242
+> type 'start'
+> type ctrl + x + a to have a better view of the code
```

Comment:
Why do we now have the `>` prompt, if we're just using english words to describe actions instead of showing?

Reply:
@babusid